pkg/operator/resourcesynccontroller/resourcesync_controller.go

```diff
@@ -137,7 +137,6 @@ func (c *ResourceSyncController) sync() error {
 	switch operatorSpec.ManagementState {
 	case operatorv1.Managed:
 	case operatorv1.Unmanaged:
-		return nil
 	case operatorv1.Removed:
```

Comment:
Does `Removed` and the `default` case make sense?  In general it seems like we always want to be in "managed" state for certain operators (and not others).

Reply:
Also `""` needs to be tolerated better IMO since the field is optional.

Reply:
I can't see "Remove" working for control plane :-)